### PR TITLE
Refactored the website

### PR DESCRIPTION
**Pull request by ratnakar5938**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [ ] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: config/content/About.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable max-len */
+export default {
+  title: 'About Us',
+  content:
+    'HackNITR 4.0 is a hackathon to encourage you to use your imagination and invention to develop cutting-edge technology-based solutions to challenges in the real world. It was put together by a large group of tech enthusiasts from Google Developer Student Clubs (GDSC), DSC NIT Rourkela, and OpenCode. The main goal is to grow our local community and give hackers an unforgettable experience. The plethora of fun and excitement is waiting for you from (___).',
```
Comment (abhibhaw) [inline config/content/About.js:5]: Get this content reviewed!!

File: config/website.js [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 module.exports = {
   pathPrefix: '/', // Prefix for all links. If you deploy your site to example.com/blog your pathPrefix should be "blog"
-  siteTitle: 'HackNITR 3.0', // Navigation and Site Title
+  siteTitle: 'HackNITR 4.0', // Navigation and Site Title
   siteTitleAlt: 'Official Website of HackNITR', // Alternative Site title for SEO
-  siteTitleShort: 'HackNITR 3.0', // short_name for manifest
+  siteTitleShort: 'HackNITR 4.0', // short_name for manifest
   siteUrl: 'https://hacknitr.tech', // Domain of your site. No trailing slash!
```
Comment (abhibhaw) [inline config/website.js:8]: .tech ko .com

Comment (abhibhaw) [reply config/website.js:8]: Also the site description

Comment (abhibhaw) [reply config/website.js:8]: Review this [page](https://github.com/dscnitrourkela/project-oregano/blob/4065cd8f390067665debdb25187b8877561f5daa/config/website.js) to check if any old links are there

**Review by abhibhaw — CHANGES_REQUESTED** [review]

File: src/styles/global.css [hunk 3]
```diff
@@ -1,6 +1,7 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Six+Caps&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans&display=swap');
```
Comment (abhibhaw) [inline src/styles/global.css:4]: Any specific reason of adding a new font?

**Review by abhibhaw — APPROVED** [review]
LGTM

File: src/styles/global.css (hunk 3 above)
Comment (ratnakar5938) [reply src/styles/global.css:4]: font for sponsor button as the same font was used in devfolio button

**Review by abhibhaw — APPROVED** [review]
lgtm

**Comment by abhibhaw** [comment]
@ratnakar5938 Start writing more meaningful commit messages!!

**Pull request merged**